certbot/docs/using.rst

```diff
@@ -751,6 +751,23 @@ time, Certbot will remember these options and apply them once again.
 Sometimes, you may encounter the need to change some of these options for future certificate renewals. To achieve this,
 you will need to perform the following steps:
 
+Certbot v2.3.0 and newer
+~~~~~~~~~~~~~~~~~~~~~~~~
+The ``certbot reconfigure`` command can be used to change a certificate's renewal options.
+This command will use the new renewal options to perform a test renewal against the Let's Encrypt staging server.
+If this is successful, the new renewal options will be saved and will apply to future renewals.
+
+As a practical example, if you were using the ``webroot`` authenticator and had relocated your website to another directory,
+you would need to change the ``--webroot-path`` to the new directory:
+
+.. code-block:: shell
+
+  certbot reconfigure --cert-name example.com --webroot-path /path/to/new/location
+
+If this command succeeds, you are done!
```

Comment:
Thanks for taking this! One thing I would add is something like "For other common options that may be updated with the "reconfigure" subcommand, run `certbot help reconfigure`." Maybe also that the certificate needs to be specified with the `--cert-name` flag, which is often the first domain on the certificate.

Reply:
Added! 

I think maybe we will want to expand the help topic to include flags like `--standalone`, `--http-01-port`, `--http-01-address`, `--https-port`, `--preferred-chain`, `--preferred-challenges` `--[no-]reuse-key`. It's kind of a bummer that there doesn't appear to be a way to include topics in plugin flags like `--webroot-path`. There might be a backwards-compat way to do this with a `kwarg` thoughl

Reply:
Good call. Adding most of those is very easy. Yeah, I think I briefly looked at adding `--webroot-path` and thought it wasn't worth the effort at the time, but I could see it being worth doing in its own PR. We could also include a note in the top matter to run `certbot help webroot` for more options or something.